griffe is not needed to run tests.
We do not have griffe packaged (yet).

--- setuptools_scm-9.2.1/pyproject.toml.orig
+++ setuptools_scm-9.2.1/pyproject.toml
@@ -71,7 +71,6 @@
   "mypy~=1.13.0", # pinned to old for python 3.8
   'typing-extensions; python_version < "3.11"',
   "wheel",
-  "griffe",
   "flake8",
 ]
 
